src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,153 @@
+package duke;
+
+import duke.command.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+public class Parser {

```

Comment:
Can consider using regex to make your parser more constomizable and less reliant on String manipulation.